README.md

```diff
@@ -4,8 +4,14 @@
 
 WARNING: if you fork this repo, github actions will run daily on it. To disable this, go to <myuser>/examples/settings/actions and Disable Actions for this repository.
 
-A repository showcasing examples of using [PyTorch](https://github.com/pytorch/pytorch):
+`pytorch/examples` is a repository showcasing examples of using [PyTorch](https://github.com/pytorch/pytorch). The goal is to have curated, short, *high quality* examples that are substantially different from each other that can be plugged into your existing work.
 
+For tutorials: https://github.com/pytorch/tutorials
+For changes to pytorch.org: https://github.com/pytorch/pytorch.github.io
+For a general model hub: https://pytorch.org/hub/ or https://huggingface.co/models
+For recipes on how to run PyTorch in production: https://github.com/facebookresearch/recipes
+For general Q&A and support: https://discuss.pytorch.org/
+## Available models
```

Comment:
Should we consider archiving some of the more outdated ones?

Reply:
Yep! Which ones do you think we should kill?

Reply:
Maybe not applicable for this rewrite, but we may want to think about how we make sure examples continue to work/are tested. High quality, to me, means they're still useful and working in the future for anyone taking a look at them.

Reply:
dcgan was outdated in 2019, fast-neural-style is pretty old now. 
from scouring the issues, the imagenet example is (was?) canonical for using DP, DDP  but it can do with some refactoring IMO. Not sure how relevant MNIST hogwild and SNLI are now.

Reply:
"High quality" to me here means showcasing best practices and associated caveats, for example what's the diff when we're using multinode vs single-node DDP

Reply:
Agreed, best practices should be there for high quality, but if the example stops working after a release, this is also not useful for folks

Reply:
All good points let's discuss this on the internal doc to see which deprecations we should prioritize